cmd/notary/tuf.go

```diff
@@ -398,7 +451,8 @@ func verify(cmd *cobra.Command, args []string) {
 		logrus.Error("notary: data not present in the trusted collection.")
 		os.Exit(1)
 	} else {
-		_, _ = os.Stdout.Write(payload)
+		_, _ = output.Write(payload)
+		output.Close()
```

Comment:
Sorry, I'm having a rough week. That defer should also be moved to the top of the function. Obviously it's never going to get executed if the write panics and it's the next line of code. Also, postcondition on the function should be output buffer is closed, in the current implementation, the output buffer may or may not be closed depending on whether there's an error or not. (it'll get cleaned up on exit anyway, but better explicit than implicit)